src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryServiceV1.java

```diff
@@ -9,10 +9,12 @@
 import net.snowflake.client.jdbc.telemetry.TelemetryClient;
 import net.snowflake.client.jdbc.telemetry.TelemetryData;
 import net.snowflake.client.jdbc.telemetry.TelemetryUtil;
+import org.apache.kafka.common.utils.AppInfoParser;
 
-import java.io.IOException;
```

Comment:
There's a lot of change sin this file. Are these related to pipe creation telemetry ?

Reply:
Most changes in this PR is to remove current telemetry that will be covered by new telemetry.